requirements.txt

```diff
@@ -14,3 +14,12 @@ python-jose==3.3.0
 flask==2.2.5
 flask-talisman==1.0.0
 dash_auth==2.0.0
+dash-leaflet==0.1.28
+
+# emission requirements
+arrow==1.2.3
+pymongo==4.4.1
+pandas==2.0.3
+future==0.18.3
+attrdict==2.0.1
+geojson==3.0.1
```

Comment:
Again, if you run this in the container defined in the `docker-compose.yml`, you will not need all these dependencies.

Reply:
Sweet, I'll nuke these lines :+1: